src/main/resources/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder/orphanedItemStrategyDescription.properties

```diff
@@ -0,0 +1,2 @@
+blurb=Computed folders can remove items immediately or leave them based on a desired retention strategy. \
+   By default, items will be removed as soon as the folder computation determines they are no longer present.
```

Comment:
```suggestion
  By default, orphaned items will be removed immediately after they are marked as orphaned, however they may also be kept based on an orphaned item retention strategy.
```

Reply:
@dwnusbaum  Hows this wording?  Avoids some of the technical words to speak generally of folders that generate their contents.